Code_experimental_design.py

```diff
@@ -1,5 +1,6 @@
 """This package contains the code for the experimental design. It includes all trials step by step.
-It only works if you combine it with the contents of the compression-code.py package (some of which are imported into this package)."""
+It only works if you combine it with the contents of the compression-code.py package (some of which are imported into this package).
+Also it is connected to the Stock files library, which contains all important images."""
 from psychopy import visual, core, event
 from PIL import Image
 import PIL
```

Comment:
i think that in order to use a function from a different script you can

from Compression_code import im (or something similar)